### ISSUE TITLE
Unable to add resource to status page

### ISSUE THREAD

**Issue by redzioch** [BetterStackHQ/terraform-provider-better-uptime#49]
I am trying to create/modify/refresh resource `betteruptime_status_page_resource`, but I get an error:

```
  # betteruptime_status_page_resource.bu_resource will be created
  + resource "betteruptime_status_page_resource" "bu_resource" {
      + availability           = (known after apply)
      + id                     = (known after apply)
      + position               = 2276
      + public_name            = "********"
      + resource_id            = 12345
      + resource_type          = "Monitor"
      + status_history         = (known after apply)
      + status_page_id         = "12345"
      + status_page_section_id = 12345
      + widget_type            = "history"
    }
```

and I get an error:

```
Error: Invalid address to set: []string{"status_history", "0", "maintenance_duration"}
```

Configuration was working for months and stopped working on Jul 4th around 4 PM GMT.

It seems that something on API side was changed and now provider returns an error whatever I am trying to do with resource.

I am using Terraform 1.3.7, BetterUptime provider version 0.3.17.

**Comment by adikus** [comment]
Thanks for the ping @redzioch, we have indeed added a new field to the `status_history` API attribute,

We'll need to add it here https://github.com/BetterStackHQ/terraform-provider-better-uptime/blob/4e74f1cc1d861fcf0af0d75d77f180ef65c4df36/internal/provider/resource_status_page_resource.go#L88-L104

**Comment by redzioch** [comment]
Yeah... I "like" (hate?) Golang being strict about variable types :)

**Comment by AleksandarGjorgievski** [comment]
This is also a blocker for us, since we have additional resources in the same state file, and we can't make any changes at all. Since the issue has been identified, is there an ETA for a new release?

**Comment by adikus** [comment]
I pushed a fix for this, it should be released under 0.3.18 in a bit
